docs/decommissioning.md

```diff
@@ -61,15 +61,15 @@ Administrators perform decommissioning operation in two approaches:
 
 1. Via Celeborn Worker REST API endpoint:
   ```shell
-  curl -X POST -d "type=Decommission" http://ip:port/exit
+  curl -X POST -H "Content-Type: application/json" -d '{"type":"Decommission"}' http://ip:port/api/v1/workers/exit
   ```
 2. Via Celeborn Master(Leader) REST API endpoint:
   ```shell
-  curl -X POST -d "type=Decommission&workers=ip_1:rpcPort:pushPort:fetchPort:replicatePort,ip_2:rpcPort:pushPort:fetchPort:replicatePort" http://ip:port/sendWorkerEvent
-  curl -X POST -d "type=DecommissionThenIdle&workers=ip_1:rpcPort:pushPort:fetchPort:replicatePort,ip_2:rpcPort:pushPort:fetchPort:replicatePort" http://ip:port/sendWorkerEvent
+  curl -X POST -H "Content-Type: application/json" -d '{"eventType":"Decommission","workers":"ip_1:rpcPort:pushPort:fetchPort:replicatePort,ip_2:rpcPort:pushPort:fetchPort:replicatePort"}' http://ip:port/api/v1/workers/events
+  curl -X POST -H "Content-Type: application/json" -d '{"eventType":"DecommissionThenIdle","workers":"ip_1:rpcPort:pushPort:fetchPort:replicatePort,ip_2:rpcPort:pushPort:fetchPort:replicatePort"}' http://ip:port/api/v1/workers/events
```

Comment:
The request body is not correct. FYI:
<img width="1685" alt="image" src="https://github.com/user-attachments/assets/3c0be644-7f0d-4f34-b3a5-4f457bfcb386">